week3/homework/src/index.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const app = express();
+const add = require('./add');
+const update = require('./update');
+const remove = require('./remove');
+const reset = require('./reset');
+const readTodo = require('./readTodo');
+const markAsDone = require('./markAsDone');
+const markAsNotDone = require('./markAsNotDone');
+
+app.use(express.json());
+
+try {
+  app.get('/todos/:id', (request, response) => {
+    readTodo(request.params.id);
+    response.status(200).json({ message: `todo with the id of ${request.params.id} is requested` });
+  });
+
+  app.post('todos/:id/:text', (request, response) => {
```

Comment:
this is not the correct format for creating todos. Please review the lecture video. You need to use the body of the requests